docs/unified/novacustom/features.md

```diff
@@ -0,0 +1,140 @@
+# Special features
+
+This document describes special firmware features specific to NovaCustom
+laptops.
+
+## Battery charge management
+
+NovaCustom laptops equipped with Dasharo firmware provide several battery
+management mechanisms to optimize the user experience and prolong the battery's
+lifespan.
+
+### Low battery boot prevention
+
+To prevent corruption of the bootloader, which may occur due to a sudden power
+loss during the OS boot process, booting the OS is blocked when the battery
+level is below 5%. When trying to boot the OS with the battery level below the
+defined threshold, the following message will be shown on the screen:
+
+![boot_block_popup](../../images/battery_block_popup.jpg)
+
+### Disconnected battery warning
+
+When the firmware encounters problems with detecting the battery, the following
+message will appear suggesting checking the physical connection between the
+battery and mainboard:
+
+![battery_connection](../../images/battery_connection.jpg)
```

Comment:
@mkopec screenshots are not possible?

Reply:
No, sreenshots don't work at that point, they are available only after the popup is dismissed :\

Reply:
ok, thx